### PR TITLE
EMP-613 - CRM14 - Defect Fix - Income section

### PR DESCRIPTION
**Pull request by gpjustice**

### CODE REVIEW

File: server/views/components/customDisplay/crm14/crm14Income.njk [hunk 1]
```diff
@@ -1,10 +1,18 @@
...
+    {% else %}
+        {% set _ = fieldsQ1.push({ label: "Do you have any proof of receipt of benefits", value: data.proofBenefits + "No (enter details of your benefits income in the next section to enable your application to be processed as a non-passported assessment)"+ '<br> <br>' + "N.B. It is in your interests to provide evidence demonstrating that you receive one of the (passported) benefits listed above, if your application is processed as a non-passported assessment and your case progresses to the Crown Court you may be liable to pay a contribution towards the cost of your case from any Capital or Equity you may have"}) %}
```
Comment (muyinatech) [inline server/views/components/customDisplay/crm14/crm14Income.njk:13]: I don't think you need + "No .."  As data.proofBenefits will be equal to "No"

Comment (gpjustice) [reply server/views/components/customDisplay/crm14/crm14Income.njk:13]: `proofBenefits` is a String, We are getting `Yes` & Empty string `" "` values, 
I’m treating empty string as `No` and applied the logic.

Comment (muyinatech) [reply server/views/components/customDisplay/crm14/crm14Income.njk:13]: Yes understood that in UAT it is in empty string, are you sure that we are not getting proofBenefits = "No" in production? If we do, we will have "No No" in the label text.

Comment (gpjustice) [reply server/views/components/customDisplay/crm14/crm14Income.njk]: Thanks, just fixed and pushed it, plz check

**Review by muyinatech — APPROVED** [review]

**Pull request merged**